notebooks/run_model_in_notebook.ipynb

```diff
@@ -15,11 +15,9 @@
    "outputs": [],
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #7.    work_dir_dict, config_path_dict = iostreamer.prepare_work_dir(forcing_filenames_list)
</pre><p><code>config_path_dict = []</code></p><p><code>input_dir_dict = []</code></p><p><code>for nc_file in forcing_filenames_list:</code></p><p><code>&nbsp;&nbsp;input_dir, output_dir, config_path = isotreamer.create_io_dir(nc_file)</code></p><p><code>&nbsp;&nbsp;config_path_dict[nc_file] = config_path</code></p><p><code>&nbsp;&nbsp;input_dir_dict[nc_file] = input_dir</code></p><p><code>&nbsp;&nbsp;print(input_dir, output_dir, config_path)</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/EcoExtreML/STEMMUS_SCOPE_Processing/pull/8/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
<p>Revised.</p>